packages/console/src/components/Table/TableEmpty.module.scss

```diff
@@ -0,0 +1,22 @@
+@use '@/scss/underscore' as _;
+
+.tableEmpty {
```

Comment:
Missing a `padding-top`.
<img width="612" alt="image" src="https://user-images.githubusercontent.com/10806653/160319917-e60f706e-a032-4e94-a43f-fe17014bd974.png">

Reply:
add 4 units padding (table row's padding is 5 units), PTAL